### PR TITLE
 Integrate Logging Application

### LINKED ISSUE #26: Integrate Logging Application
- Read Integrate Logging Application
- To the server side code , integrate Logging Applications.
- Read Running Node.js application in VSCode
- Run the NodeJs application and do a basic sanity
- Verify whether the server is getting started without any error
- Incase of any issues , Read Debugging Node.js application in VSCode and resolve the issue.
- Read Processing an Issue
- Read Creating a Commit
- Process the Issue you created and commit the changes
- Read Completing an Issue and creating a Merge Request
- Push your branch containing your contribution and add a CodeOwner as reviewer
- Confirm that the CodeOwner merged your contribution

### PR DESCRIPTION
**Pull request by MFMentopolis**
Fixes #26 Integrate Logging Application

### CODE REVIEW

File: server/controllers/OperationClient.js [hunk 1]
```diff
@@ -2,83 +2,101 @@
 
 var utils = require('../utils/writer.js');
 var OperationClient = require('../service/OperationClientService');
+var oamLogService = require('onf-core-model-ap/applicationPattern/services/OamLogService');
+var executionAndTraceService = require('onf-core-model-ap/applicationPattern/services/ExecutionAndTraceService');
```
Comment (PrathibaJee) [inline server/controllers/OperationClient.js]: ExecutionAndTraceService is for "Basic and Individual" services. So , this unused require statement in the controller file of the OAM service is not required. Please remove this.

**Review by PrathibaJee — CHANGES_REQUESTED** [review]
Many thanks for integrating the logging mechanism to the basic , individual and OAM services.

- Basic and Individual services looks fine.
- In the OAM service , I have the following comment , 
        - Currently , you have updated the OAMlogging services to the OperationClient.js.
        - Similarly , this needs to updated in the following files as well , as this files also related to the OAM services, 
                1. Core.js , 
                2. HttpClient.js
                3. HttpServer.js
                4. OperationServer.js
                5. TcpClient.js
                6. TcpServer.js

Kindly please let me know incase of any queries. 
Many thanks.

File: server/controllers/OperationClient.js (hunk 1 above)
Comment (MFMentopolis) [reply server/controllers/OperationClient.js]: OK, thank you very much for your pointers!

**Review by PrathibaJee — APPROVED** [review]
Many thanks for resolving the review comments.
Changes are as expected for the activity "Integrate Logging Application".
Approving the changes.

Kindly please resolve the merge conflict and merge the issue to develop branch. Thanks.

**Pull request merged**